### DISCUSSION TITLE
How can I use react-table for infinite scrolling?

### DISCUSSION THREAD

**Discussion by nate15** [TanStack/table#3995] [Q&A] [answered]
I want to use react-table for infinite scrolling, but I couldn't find any resource. some say use react window or react virtualized but they don't show how.

**Comment by tannerlinsley** [comment, marked as answer]
You’re probably looking for https://TanStack.com/virtual

Tanner Linsley

**Reply by scouty-io** [reply]
Is there an example for this or just a link to the home landing page ?
